homeassistant/components/go2rtc/__init__.py

```diff
@@ -1,20 +1,37 @@
 """The go2rtc component."""
 
-from go2rtc_client import Go2RtcClient, WebRTCSdpOffer
+import logging
+
+from go2rtc_client import Go2RtcRestClient
+from go2rtc_client.ws import (
+    Go2RtcWsClient,
+    ReceiveMessages,
+    WebRTCAnswer,
+    WebRTCCandidate,
+    WebRTCOffer,
+    WsError,
+)
 
 from homeassistant.components.camera import Camera
 from homeassistant.components.camera.webrtc import (
     CameraWebRTCProvider,
+    WebRTCAnswer as HAWebRTCAnswer,
```

Comment:
Side note: Normally we only import from the integration package module and not from submodules.